lisdf/parsing/pddl_j.py

```diff
@@ -47,7 +47,7 @@ def transform(
 
 class PDDLVisitor(Transformer):
     def __init__(
-        self, sdf: C.LISDF, domain: C.PDDLDomain, problem: C.PDDLProblem
```

Comment:
nit: is `lisdf` a better variable name? I can't remember the details of the lisdf representation...